libs/expo/betterangels/src/lib/helpers/generatePublicNote.ts

```diff
@@ -68,9 +66,20 @@ export default function generatePublicNote(watchedValues: IWatchedValue) {
     if (item.service === ServiceEnum.Other) {
       return item.customService;
     }
-    return item.service;
+    return enumDisplayServices[item.service];
   });
 
+  const updatedP =
+    requestedServicesArray.length > 0
+      ? 'P - Follow up with client regarding requested ' +
+        requestedServicesArray.slice(0, -1).join(', ').toLowerCase() +
+        (requestedServicesArray.length > 1 ? ', and ' : '') +
```

Comment:
i'm not sure the comma ` , ` is needed there (before the `and`). 
this will yield the `pet food, and hygiene` as opposed to `pet food and hygiene`